src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -295,6 +295,43 @@ void SubgraphBaseTest::configure_model() {
     function = p.build();
 }
 
+bool SubgraphBaseTest::is_allowed_convertion(ov::element::Type from, ov::element::Type to) {
+// based on logic from /src/common/transformations/src/transformations/convert_precision.cpp
+    if ((from != to) &&\
```

Comment:
let's split this long condition to parts? for example, if `from.is_real() && to.is_intergral`